docs/package.json

```diff
@@ -11,8 +11,8 @@
     "build:notfound": "node scripts/generate-notfound.js",
     "build:docs": "node scripts/generate-documentation.js",
     "build:blog": "node scripts/generate-blog.js",
-    "build": "yarn build:assets && yarn build:home && yarn build:blog && yarn build:docs && yarn build:notfound",
-    "watch": "cross-env WATCH=1 yarn start",
+    "build": "npm run build:assets && npm run build:home && npm run build:blog && npm",
```

Comment:
looks like this line is missing the `build:docs` and `build:notfound` bits at the end